cli/waiter/cli.py

```diff
@@ -16,8 +16,10 @@
 
 subparsers = parser.add_subparsers(dest='action')
 
+create_or_update = create.register(subparsers.add_parser)
 actions = {
-    'create': create.register(subparsers.add_parser),
+    'create': create_or_update,
+    'update': create_or_update,
```

Comment:
Is it confusing that update works exactly like create? Maybe update should only be allowed on existing tokens?

Reply:
If we do that, then shouldn't we make create only work for new tokens? The downside is that then I'd have to remember which command to use, even though they do the exact same thing. I think I prefer letting either command do both, and users can still choose to only use create for new and update for existing.